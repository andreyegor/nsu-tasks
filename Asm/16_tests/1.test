zoo
ant
bird
elephant
cat