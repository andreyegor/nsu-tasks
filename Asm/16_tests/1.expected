ant
bird
cat
elephant
zoo